src/tests/ftest/util/server_utils_params.py

```diff
@@ -495,6 +494,7 @@ def __init__(self, base_namespace, index, provider=None, max_storage_tiers=MAX_S
             "ABT_ENV_MAX_NUM_XSTREAMS=100",
             "ABT_MAX_NUM_XSTREAMS=100",
             "DAOS_MD_CAP=1024",
+            "DAOS_POOL_RF=4",
```

Comment:
@daltonbohning would you help point me how i could modify this parts? I notice if i moved DAOS_POOL_RF=4 to here, some tests, eg rebuild won't inherit this proper：
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-15196/16/artifact/Functional%20Hardware%20Medium%20Verbs%20Provider/daos_test/suite.py/daos_configs.wolf-217/daos_server.yml/*view*/

But i want to specify this in dmg pool query test, all other tests should be DAOS_POOL_RF=4, it looks a messy to me.

Reply:
I think the problem is this is the default set of `env_vars`, but since https://github.com/daos-stack/daos/blob/master/src/tests/ftest/daos_test/suite.yaml#L51-L57 defines `env_vars` these defaults aren't used.

@phender Is this the intended behavior?

Reply:
Just curious why CRT_EVENT_DELAY and DAOS_POOL_RF set at different places?